client/my-sites/marketing/do-it-for-me/site-build-showcase.tsx

```diff
@@ -33,16 +33,16 @@ const Container = styled.div`
 		opacity: 0;
 	}
 
-	> div:nth-child( 2 ) {
+	> div:nth-of-type( 2 ) {
```

Comment:
Decided to include this change in this PR.
Using `nth-child` caused this error:
<img width="1169" alt="image" src="https://user-images.githubusercontent.com/5436027/183038741-a91f77a2-183a-4dbf-a148-166116295199.png">